### ISSUE TITLE
crash, might be compatibility related (resharper)

### ISSUE THREAD

**Issue by jimmymain** [DotNetAnalyzers/StyleCopAnalyzers#1875]
![image](https://cloud.githubusercontent.com/assets/2555050/11495748/57f91d42-9816-11e5-86ce-ac42e897032e.png)

I am getting this error regularly, and never used to.
I have tried debugging it, but when it runs in the sandbox (regular F5 debug) resharper is not enabled along with any other plugins, and I cannot reproduce the issue.

How do I get the stack trace out in the regular environment, then I can try and track it down.

It is occurring, for instance when a constructor does not have correct documentation.

**Comment by sharwell** [comment]
I'm not sure it's currently possible.

One thing you can try is opening two copies of Visual Studio, and attaching the debugger from one to the other. This will allow you to debug your normal instance of Visual Studio as opposed to the sandbox you get when you <kbd>F5</kbd> on the project.

**Comment by jimmymain** [comment]
I reset all my settings, and it seems to be ok now.
weird, I don't know if you want to close it, and I can let you know if it happens again. It seemed to affect quite a lot of the stylecop quick fixes. I think more an issue with corrupt configuration in visual studio itself rather than a bug in the analyzers.

**Comment by sharwell** [comment]
We'll close it for now, but if you reproduce it we'll open it right back up. :+1:

**Comment by NikolayIT** [comment]
I've just got the same error with VS 2015 Update 1, ReSharper 10 and StyleCopAnalyzers 1.0.0-rc1

![error](https://cloud.githubusercontent.com/assets/3106986/11504594/d7d74dd0-984e-11e5-8fa6-43b0c28658b2.png)

**Comment by sharwell** [comment]
@NikolayIT 

1. Does this occur every time you try to run SA1133 fixer?
1. If the answer to 1 is yes (meaning you can reproduce it), does it still occur if you disable ReSharper (but no other changes)?

**Comment by NikolayIT** [comment]
Yes, it does. I can reproduce it every single time. Even with ReSharper disabled.

To be more specific it happens when I click on the light bulb icon.

I've recorded a video: https://www.dropbox.com/s/0w7109ze2oom8yz/VS2015Update1StyleCopAnalyzersBug.mp4?dl=0

**Comment by sharwell** [comment]
@vweijsters Any ideas off the top of your head, before we dig into this?

**Comment by jimmymain** [comment]
Hi Guys,

I had the same setup, but it did seem ok once I reset all my settings.
I tried to debug it but could not reproduce in the debugger at all.

I tried to attach the debugger, no luck there either.

I really don't think it's an issue with the analyser.
My feeling is that its one of those combination bugs. It's going to be
difficult to track down. Happy to help if anyone has suggestions.

**Comment by vbfox** [comment]
I just entered a bug (#1878) that is in SA1133 too @NikolayIT so maybe it's the same. Cc @sharwell

**Comment by pdelvo** [comment]
@jimmymain Could you try to attach a visual studio instance to your crashing one and break on all exceptions? The code fix is relatively complicated and it will be very hard to track this down without some more information.

Im not sure, maybe this will also get logged into visual studio's Activity log. Maybe you have to start visual studio with the [Log switch](https://msdn.microsoft.com/en-us/library/ms241272.aspx).

**Comment by jimmymain** [comment]
Hi Dennis

I will try - but it stopped recurring so I might not be able to reproduce
it.
